analysis/analysis_test.go

```diff
@@ -208,10 +213,15 @@ func TestApplicationAnalysis(t *testing.T) {
 					}
 					if len(got.Incidents) != len(expected.Incidents) {
 						t.Errorf("Different amount of incidents error. Got %d, expected %d.", len(got.Incidents), len(expected.Incidents))
-						t.Error("Got:")
-						pp.Println(got.Incidents)
-						t.Error("Expected:")
-						pp.Println(expected.Incidents)
+						missing, unexpected := getIncidentsDiff(expected.Incidents, got.Incidents)
+					    for _, incident := range missing {
+							fmt.Println("Expected incident not found:")
+							pp.Println(incident)
+					    }
+					    for _, incident := range unexpected {
+							fmt.Println("Unexpected incident found:")
+							pp.Println(incident)
+					    }
```

Comment:
Can we not print all fields of an incident just to keep it concise, I think just file URI and line number is enough